receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/filesystem_scraper_others.go

```diff
@@ -28,9 +28,19 @@ const fileSystemStatesLen = 2
 func (s *scraper) recordFileSystemUsageMetric(now pdata.Timestamp, deviceUsages []*deviceUsage) {
 	for _, deviceUsage := range deviceUsages {
 		s.mb.RecordSystemFilesystemUsageDataPoint(
-			now, int64(deviceUsage.usage.Used), deviceUsage.partition.Device, getMountMode(deviceUsage.partition.Opts), deviceUsage.partition.Mountpoint, deviceUsage.partition.Fstype, metadata.AttributeState.Used)
+			now, int64(deviceUsage.usage.Used),
+			deviceUsage.partition.Device, getMountMode(deviceUsage.partition.Opts),
+			deviceUsage.partition.Mountpoint, deviceUsage.partition.Fstype,
+			metadata.AttributeState.Used)
 		s.mb.RecordSystemFilesystemUsageDataPoint(
-			now, int64(deviceUsage.usage.Free), deviceUsage.partition.Device, getMountMode(deviceUsage.partition.Opts), deviceUsage.partition.Mountpoint, deviceUsage.partition.Fstype, metadata.AttributeState.Free)
+			now, int64(deviceUsage.usage.Free),
+			deviceUsage.partition.Device, getMountMode(deviceUsage.partition.Opts),
+			deviceUsage.partition.Mountpoint, deviceUsage.partition.Fstype,
+			metadata.AttributeState.Free)
+		s.mb.RecordSystemFilesystemUtilizationDataPoint(
+			now, deviceUsage.usage.UsedPercent,
```

Comment:
I think `usage.UsedPercent` is still reported in percents

Reply:
You are right, I will fix it. Thank you